volatility3/framework/plugins/timeliner.py

```diff
@@ -171,18 +172,22 @@ def _generator(
                                     TimeLinerType.CHANGED,
                                     renderers.NotApplicableValue(),
                                 ),
+                                times.get(
+                                    TimeLinerType.BOOTTIME,
+                                    renderers.NotApplicableValue(),
+                                ),
                             ],
                         )
                     )
 
                     # Write each entry because the body file doesn't need to be sorted
                     if fp:
                         times = self.timeline[(plugin_name, item)]
-                        # Body format is: MD5|name|inode|mode_as_string|UID|GID|size|atime|mtime|ctime|crtime
+                        # Body format is: MD5|name|inode|mode_as_string|UID|GID|size|atime|mtime|ctime|crtime|boottime
```

Comment:
Other fields are already defined here for the body format?  Doesn't seem like diverging from the standard is a good idea...  5:S
https://wiki.sleuthkit.org/index.php?title=Body_file

Reply:
done